frontend/components/FileUploader/FileUploader.tsx

```diff
@@ -21,7 +22,7 @@ type ISupportedGraphicNames = Extract<
 >;
 
 interface IFileUploaderProps {
-  graphicName: ISupportedGraphicNames;
+  graphicNames: ISupportedGraphicNames[];
```

Comment:
This change breaks the current component API which forces us to change every instance where we use it. In some cases this makes sense but here showing multiple graphics is the exception not the norm (i.e. Scripts uploader is the only place we need this functionality)

I'd suggest we instead extend the `graphicName` prop be `ISupportedGraphicNames | ISupportedGraphicNames[]`. That way it still works where its currently used without changes and supports the new requirement of displaying multiple graphics when needed.

Reply:
Good point, that makes sense

Reply:
I think the prop name `graphicNames` (plural) is more accurate, but the union type makes a lot of sense. Will restore `graphicName` (singular) as the propname for now, we can update once @ghernandez345 has a chance to weigh in tomorrow.